src/DynamoCoreWpf/ViewModels/Search/NodeAutoCompleteSearchViewModel.cs

```diff
@@ -837,8 +839,26 @@ internal void AddCluster(ClusterResultItem ClusterResultItem)
                 catch (Exception) { }
             });
 
+            var clusterNodesModel = clusterMapping.Values.ToList();
+            clusterNodesModel.ForEach(nodeInCluster => nodeInCluster?.NodeModel?.AllConnectors?.ToList().ForEach(connector =>
+            {
+                if (connector != null) connector.IsHidden = true;
+            }));
+
+            Action finalizer = () =>
```

Comment:
Some comments? I assume temporarily make connectors not visible. Also worth testing in DynamoRevit context, I remember seeing some crash like this if not grabbing the dispatcher from the current thread or so, things usually working great in Sandbox

Reply:
It's fine, we are using this approach to postpone execution on DispatcherPriority.ApplicationIdle in many contexts.
Will add comments.